ml/src/main/scala/org/apache/spark/ml/h2o/param/H2OMOJOPipelineModelParams.scala

```diff
@@ -27,24 +27,33 @@ trait H2OMOJOPipelineModelParams extends Params {
   //
   // Param definitions
   //
-  final val namedMojoOutputColumns: Param[Boolean] = new BooleanParam(this, "namedMojoOutputColumns", "Mojo Output is not stored" +
+  private val namedMojoOutputColumns: Param[Boolean] = new BooleanParam(this, "namedMojoOutputColumns", "Mojo Output is not stored" +
     " in the array but in the properly named columns")
+  private val featuresCols: StringArrayParam = new StringArrayParam(this, "featuresCols", "Name of feature columns")
 
   //
   // Default values
   //
-  setDefault(namedMojoOutputColumns -> true)
+  setDefault(
+    namedMojoOutputColumns -> true,
+    featuresCols -> Array.empty[String]
+  )
 
   //
   // Getters
   //
   /** @group getParam */
   def getNamedMojoOutputColumns() = $(namedMojoOutputColumns)
 
+  /** @group getParam */
+  def getFeaturesCols(): Array[String] = $(featuresCols)
+
   //
   // Setters
   //
   /** @group setParam */
   def setNamedMojoOutputColumns(value: Boolean): this.type = set(namedMojoOutputColumns, value)
 
+  /** @group setParam */
+  def setFeaturesCols(cols: Array[String]): this.type = set(featuresCols, cols)
```

Comment:
I'm not sure whether we should specify a setter for the property.

Reply:
On the model, I would keep for `featuresCols` just getter